### DISCUSSION TITLE
👻 Answer this discussion to get Galaxy Brain

### DISCUSSION THREAD

**Discussion by zerosrat** [drknzz/GitHub-Achievements#102] [Q&A] [answered]
Here we go

**Comment by Fuzzworth** [comment, marked as answer]
Lets gooo

**Reply by NahuelOrselli** [reply]
hey Fuzz! reply in my discussion

**Reply by the1Riddle** [reply]
let's jump into it

**Reply by the-coder-o** [reply]
hello how are you today?

**Comment by NahuelOrselli** [comment]
Go go go go

**Reply by jeffreyjackson** [reply]
This is amazing!

**Comment by solomonchidera** [comment]
Let's gooo

**Reply by jeffreyjackson** [reply]
Nice work everyone!

**Reply by solomonchidera** [reply]
someone should mark my comment as answer

**Comment by commswithsaqlain** [comment]
Lets go

**Comment by leodisarli** [comment]
Lets go

**Comment by EliasAfara** [comment]
lets go

**Reply by commswithsaqlain** [reply]
lets goo

**Comment by Jikaao** [comment]
Answering, Thanks!

**Comment by EliasAfara** [comment]
Guys do you want to keep your Github contribution steak alive without making commits personally ?
Check out my new project [contribution-streak-maintainer](https://www.npmjs.com/package/contribution-streak-maintainer), it's an npm package, you only need to create a yml file and generate a Github token, very simple procedure. 
@jeffreyjackson @zerosrat @leodisarli @Fuzzworth @Jikaao @MuhammadSaqlain143 @SolomonChidera

**Comment by bnjmnzh** [comment]
Very cool resource!

**Reply by commswithsaqlain** [reply]
yeah

**Comment by commswithsaqlain** [comment]
yes

**Comment by the1Riddle** [comment]
ready

**Comment by Git-SalahEdd** [comment]
Lets gooo

**Comment by jinDeHao** [comment]
Let's get it

**Comment by droffilc1** [comment]
> Here we go

Let's go!

**Comment by enteralastname** [comment]
Let's go!!

**Comment by karakuik** [comment]
Lets gooo

**Comment by dankwx** [comment]
hey guys !!

**Comment by RamakrushnaBiswal** [comment]
let's go

**Comment by wulfwolf** [comment]
go

**Comment by noxs1d** [comment]
lets go i wanna too

**Reply by the-coder-o** [reply]
gooo

**Comment by the-coder-o** [comment]
gooo brarthers

**Reply by the-coder-o** [reply]
yeeee

**Comment by devIgor1** [comment]
Lets gooo
